samples/nrf54h20/idle_relocated_tcm/README.rst

```diff
@@ -0,0 +1,273 @@
+
+.. contents::
+   :local:
+   :depth: 2
+
+The test benchmarks the idle behavior of an application that runs on multiple cores.
+It demonstrates a radio loader pattern where the radio core firmware is loaded from MRAM into Tightly Coupled Memory (TCM) at runtime.
+
+Requirements
+************
+
+The test supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+Overview
+********
+
+This test demonstrates how to build a multicore idle application with :ref:`configuration_system_overview_sysbuild` using a two-stage boot process for the radio core:
+
+* Radio Loader - A small bootloader that runs on the radio core, copies firmware from MRAM to TCM, and jumps to it.
+* Remote Firmware - The actual application that runs from TCM after being loaded.
+
+The test automatically relocates the remote firmware binary to the correct MRAM address during build time, ensuring it can be loaded by the radio loader.
+
+Architecture
+============
+
+The system uses the memory layout compatible with the MCUboot bootloader.
+By default, the system uses the MCUboot in the direct-xip mode in the merged slot configuration (no file suffix used).
+To configure the system to use the MCUboot in the direct-xip mode in the split slot configuration, build the test with the ``split_slot`` file suffix.
+
+Memory map configuration
+========================
+
+The memory maps are defined in the :file:`common` directory and are shared between all images to ensure consistency.
+
+The following changes are included in the project overlay files:
+
+.. tabs::
+
+   .. group-tab:: Merged slot configuration
+
+      1. Entire Radio TCM RAM region is used for the remote firmware (:file:`common/memory_map_ram_cpurad.dtsi`).
+      #. A small part of Global RAM is used for the radio loader for the firmware relocation time (:file:`common/memory_map.dtsi`).
+      #. The MRAM is partitioned for the MCUboot, application, and radio core images (:file:`common/memory_map.dtsi`).
+
+   .. group-tab:: Split slot configuration
+
+      1. Entire Radio TCM RAM region is used for the remote firmware (:file:`common/memory_map_ram_cpurad.dtsi`).
+      #. A small part of Global RAM is used for the radio loader for the firmware relocation time (:file:`common/memory_map_split_slot.dtsi`).
+      #. The MRAM is partitioned for the MCUboot, application, and radio core images (:file:`common/memory_map_split_slot.dtsi`).
+      #. The ``pm_ramfunc`` DTS node is relocated to the end of the RAM block but before the MCUboot trailer (:file:`common/memory_map_ram_pm_cpurad.dtsi`) to make the image link properly.
+
+Enabling the Radio Loader
+*************************
+
+The radio loader is automatically added to the build when you enable it in sysbuild configuration.
+
+Set the :kconfig-option:`SB_CONFIG_NRF_RADIO_LOADER` to ``y`` in the :file:`sysbuild.conf` file.
```

Comment:
```suggestion
Set the ``SB_CONFIG_NRF_RADIO_LOADER`` Kconfig option to ``y`` in the :file:`sysbuild.conf` file.
```
The doc build does not seem to like the :kconfig-option: here. And the option is not found in the Kconfig reference. Is this correct?